packages/netdog/write-network-status.service

```diff
@@ -1,17 +1,20 @@
 [Unit]
 Description=Write network status
+DefaultDependencies=no
 # Block manual interactions with this service, since it could leave the system in an
 # unexpected state
 RefuseManualStart=true
 RefuseManualStop=true
-Before=early-boot-config.service
+Before=early-boot-config.service network-online.target
```

Comment:
I had to add this explicit `Before` annotation, otherwise I get weird systemd ordering that write-network-status happens after network-online.target:
```
network-online.target @1.979s
└─systemd-networkd-wait-online.service @1.254s +722ms
  └─systemd-networkd.service @1.189s +61ms
    └─network-pre.target @1.186s
...
```

With this change, the write-network-status.service happens before the network-online.target.
```
network-online.target @1.788s
└─write-network-status.service @1.753s +33ms      <-- listed as dependency
  └─systemd-resolved.service @2.669s +217ms
    └─systemd-sysctl.service @753ms +22ms
...
```